packages/service-worker-mock/models/Headers.js

```diff
@@ -2,10 +2,19 @@
 
 class Headers {
   constructor(meta) {
+    if (typeof meta === 'undefined') {
+      // https://github.com/GoogleChrome/workbox/issues/1461
+      throw new TypeError('Constructing headers with an undefined argument fails in '
```

Comment:
Bummer that we have to create an incorrectly specced mock

Reply:
Having said that, we could make it a warning instead of an error...